### PR TITLE
Containerize/dockerize the app, add BASE_PATH support

### PR DESCRIPTION
**Pull request by pjanik**
[[#184539868]](https://www.pivotaltracker.com/story/show/184539868)

This PR containerizes the app, so we can deploy it to AWS Fargate.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM node:10
+
+WORKDIR /CEASAR-server
+COPY package*.json ./
+
+RUN npm install
```
Comment (dougmartin) [inline Dockerfile:6]: Maybe change this to `RUN npm ci` so it uses the lock file versions?

File: docker/dev/run.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+npm install
```
Comment (dougmartin) [inline docker/dev/run.sh:3]: Maybe also change this to `npm ci`?

**Review by dougmartin — APPROVED** [review]
Looks good :+1:   Just had suggestions for using `npm ci` instead of `npm install`.

**Comment by pjanik** [comment]
Thanks, it makes sense, I've just fixed that in the subsequent PR.

**Pull request merged**